### PR TITLE
Fix #111052 privatenote.co.kr

### LINKED ISSUE #111052: privatenote.co.kr
### Issue URL (Incorrect Blocking)
 [https://privatenote.co.kr/main_studying/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fprivatenote.co.kr%2Fmain_studying%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Mac v2.7.0.1094 release
Browser: | Safari
Stealth mode: | disabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base
Userscripts: | disabled

### PR DESCRIPTION
**Pull request by piquark6046**
https://github.com/AdguardTeam/AdguardFilters/issues/111052

### CODE REVIEW

File: SpywareFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -990,10 +990,10 @@ nnmclub.ro,nnm-club.t0r.xyz,nnm-club-me.appspot.com,nnm-club.lib,nnm-club.me,nnm
 @@||google-analytics.com/gtm/js$domain=jump.com.tr
 !
 ! Fixing broken web sites
-ciokorea.com,adpiamall.com,live.joins.com,naver.com,jbe.go.kr#%#//scriptlet('set-constant', 'window.wcs_add', 'noopFunc')
-ciokorea.com,adpiamall.com,live.joins.com,naver.com,jbe.go.kr#%#//scriptlet('set-constant', 'window.wcs_do', 'noopFunc')
+privatenote.co.kr,ciokorea.com,adpiamall.com,live.joins.com,naver.com,jbe.go.kr#%#//scriptlet('set-constant', 'window.wcs_add', 'noopFunc')
+privatenote.co.kr,ciokorea.com,adpiamall.com,live.joins.com,naver.com,jbe.go.kr#%#//scriptlet('set-constant', 'window.wcs_do', 'noopFunc')
```
Comment (zloyden) [inline SpywareFilter/sections/whitelist.txt:994]: Have you checked these scriptlets? I still need the exception rule.

Comment (piquark6046) [reply SpywareFilter/sections/whitelist.txt:994]: Yes. I tested on Mozilla Firefox with AdGuard Browser Extension.
An unblocking rule can be founded at line 996.

Comment (zloyden) [reply SpywareFilter/sections/whitelist.txt:994]: I meant that on my end those scriplets don't work. It's better to just do an exception rule.

Comment (piquark6046) [reply SpywareFilter/sections/whitelist.txt:994]: Okay.

Comment (piquark6046) [reply SpywareFilter/sections/whitelist.txt]: @zloyden

Comment (zloyden) [reply SpywareFilter/sections/whitelist.txt]: I see, but it looks like they don't work.
1. Remove these scriptlets.
2. Leave the exceptional rule without hint.
```
@@||wcs.naver.net/wcslog.js$domain=ciokorea.com|adpiamall.com|live.joins.com|naver.com|jbe.go.kr|privatenote.co.kr
```

**Review by zloyden — APPROVED** [review]

File: SpywareFilter/sections/whitelist.txt (hunk 1 above)
Comment (zloyden) [reply SpywareFilter/sections/whitelist.txt]: I've done it here.

Comment (piquark6046) [reply SpywareFilter/sections/whitelist.txt]: I cannot understand your meaning.
I think the website works correctly.

<details>
<summary>Firefox with AdGuard Browser Extension</summary>

Screenshot0
Screenshot2
Screenshot3

</details>

Comment (zloyden) [reply SpywareFilter/sections/whitelist.txt]: Just look at my commit https://github.com/AdguardTeam/AdguardFilters/pull/111085/commits/62860b20c1b660fcc24719bab0a40bc360cd2a59 and I hope you will understand it :)

**Pull request merged**